src/components/02-buttons/buttons.jsx

```diff
@@ -27,16 +27,16 @@
  *   // Multiple class usage
  *   const config = {
  *      class: "usa-button--outline usa-button--inverse",
- *      autofocus: "true", //boolean value only
- *      disabled: "false", //boolean value only
+ *      autofocus: true, //boolean value only
+ *      disabled: false, //boolean value only
  *      name: "Button",
  *      type: "submit",
  *      value: "",
  *      innerText: "Button",
  *   }
  * @returns {HTMLButtonElement} The JSX HTML button
  */
-exports.Button = function(config) {
+exports.Buttons = function (config) {
```

Comment:
It looks like this outputs a single `<button>`, which might get confusing if this component is plural `Buttons`.

Reply:
You're right I'll switch it back.